src/Tools/dotnet-dump/dotnet-dump.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Microsoft.Diagnostic.Tools.Dump</RootNamespace>
     <Description>Diagnostic dump collect and analyze tool</Description>
     <PackageTags>Diagnostic</PackageTags>
+    <NeedsPublishing>true</NeedsPublishing>
```

Comment:
Why does just dotnet-dump need NeedsPublishing set? What about dotnet-sos? The rest of the packages.

Reply:
Because SOS.Hosting pretty much depends on SOS copying to the bin folder to work. I may look again at this today. It didn't make any sense to me.